api/agent/pure_runner.go

```diff
@@ -669,7 +669,7 @@ func (pr *pureRunner) Engage(engagement runner.RunnerProtocol_EngageServer) erro
 
 	// if we didn't respond with TooBusy, then this means the request
 	// was processed.
-	if err != models.ErrCallTimeoutServerBusy {
+	if err != nil {
```

Comment:
this will give us something more along the lines of 'requests received' vs 'requests handled' -- we probably want both, but at least this change makes requestsHandled misleading from its original definition (I am not sure of the existence of the other)

Reply:
Hmm, I'm not sure what the right thing to do is in that case. I was trying to account for the case where we get an error from the RunnerPool (other than models.ErrCallTimeoutServerBusy) -- in which case we never get as far as handling the request. Any suggestions of how to account for that without skewing this for other cases?

Reply:
ah, I see. it may be wise to put this specific statistic in call.End for pureRunner, but I'm not 100%. at least, call.End is only reached in pure runner if the task calls Submit, at which point we either ran it and it blew up or it worked. may want to consult @skinowski